.circleci/config.yml

```diff
@@ -124,14 +124,37 @@ jobs:
             echo "spring-boot.version=${spring_boot_release}"
 
             pushd /tmp/testsuite
-            oc policy add-role-to-user view -z default 
+            oc policy add-role-to-user view -z default
             mvn -s ~/project/.circleci/settings.xml clean verify -Popenshift -Dspring-boot.bom.version=${bom_version} -Dspring-boot.version=${spring_boot_release}
             popd
 
           else
             echo "The BOM version is invalid!"
             exit 1
           fi
+  release:
+    docker:
+    - image: circleci/openjdk:8-jdk-browsers
+
+    steps:
+    - checkout
+
+    - run:
+        name: Perform release
+        command: |
+          tagRE='release-(.*)'
```

Comment:
We might want to make this more precise in future versions.

Reply:
Indeed